apps/arena/lib/arena/configuration.ex

```diff
@@ -29,7 +29,11 @@ defmodule Arena.Configuration do
 
     Jason.decode!(payload.body, [{:keys, :atoms}])
     |> Map.update!(:map, fn maps ->
-      map = Enum.random(maps)
+      map =
```

Comment:
This seems ok for now, It would be better to send the maps already filtered but I'm not sure what are we going to do with the maps in the client, so, I think this is ok